### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sumindar**
Some updations.
Thanks

### CODE REVIEW

**Review by srinath1412001 — COMMENTED** [review]
hacktoberfest

File: README.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ This playground doesn't include any android and Appwrite best practices but rath
 * Make sure android studio is installed and that your android environment is set for running android projects.
 * You have a project created in AppWrite using console and note down the project ID.
 * Create the collection and documents under it. Assign the readable permission for both collection and documents .
-* Note the colection ID and Document ID as it will be used later.
-* Create a user in AppWrite using appwrite console and note down its email and password.  
+* Note the collection ID and Document ID as it will be used later.
+* Create a user in AppWrite using AppWrite console and note down its email and password.
```
Comment (eldadfux) [inline README.md:21]: Appwrite

**Comment by eldadfux** [comment]
@srinath1412001 appreciate the help, we decided grammar/typo PRs are OK as long as they are correct and not forced.

**Comment by srinath1412001** [comment]
> @srinath1412001 appreciate the help, we decided grammar/typo PRs are OK as long as they are correct and not forced.

Alright @eldadfux . I'll keep that in mind.

**Comment by srinath1412001** [comment]
Hey @Sumindar,
I'm sorry if you felt I was being rule.

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Comment by Sumindar** [comment]
Thanks a lot and sorry. I will try my best to contribute in a better manner.
Thanks you!!

**Pull request merged**